lib/msal-core/src/UserAgentApplication.ts

```diff
@@ -680,11 +694,14 @@ export class UserAgentApplication {
                 serverAuthenticationRequest.populateQueryParams(account, null, adalIdTokenObject, true);
             }
 
+            this.logger.verbose("Query parameters populated");
```

Comment:
Instead of this, I would add an `else` to the `if/else if` directly above then add a statement indicating no additional query parameters added.

Reply:
Noted. Changes made.